test/e2e/logs/log-namespace.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+SRC=$(cd $(dirname "$0"); pwd)
+source "${SRC}/../include.sh"
+
+cd $file_path/log-namespace/
+
+LOG_PATH_PREFIX="${SRC}/__log-namespace__"
+
+rm -rf "${LOG_PATH_PREFIX}"
+mkdir "${LOG_PATH_PREFIX}"
+
+$pm2 start echo.js --namespace e2e-test-log-namespace
+
+LOG_FILE_BASELINE="${LOG_PATH_PREFIX}/baseline-out.log"
+$pm2 logs e2e-test-log-namespace > $LOG_FILE_BASELINE & # backgrounded - will be stopped by `$pm2 delete all`
```

Comment:
I haven't seen any e2e (or programmatic tests) for the `pm2 logs` command, in order to mimic those. So I'm shooting a bit in the dark with this approach of backgrounding the process.